server/updaters.coffee

```diff
@@ -191,5 +191,12 @@ module.exports = ->
         incident.modifiedDate = new Date()
         Incidents.update(_id: incident._id, incident)
 
+  console.log "Removing incidents with invalid dates..."
+  invalidDateIncidents = []
+  Incidents.find().map (incident) ->
+    if incident.dateRange.end < incident.dateRange.start
+      invalidDateIncidents.push(incident._id)
+  console.log "Removing #{invalidDateIncidents.length} incidents"
+  Incidents.remove(_id: $in: invalidDateIncidents)
```

Comment:
The incidents are kind of all or none when it comes to the location, count and date info. If they're missing one those those fields we can't use them for anything, so I'm alright with removing the full incident. Also, these are mostly or all auto-generated so we'll could eventually regenerate them after the processing code has been updated.